apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/ManifestsList/CategoriesList/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useMemo } from "react";
+import { FlashList } from "@shopify/flash-list";
+import { useQuery } from "@tanstack/react-query";
+import { Box } from "@ledgerhq/native-ui";
+import Badge from "./Badge";
+
+const categories = [
```

Comment:
[No Action] Would there be a way in the future to get those categories dynamically from DappRadar? (Maybe not) - Would be nice not to have to handle that manually.

Reply:
Everything in this PR is mocked data, the actual categories from the api are coming from DappRadar